proposals/html-module-spec-changes.md

```diff
@@ -1,31 +1,29 @@
-This is a list of spec areas that will need to be changed to implement our [HTML Modules proposal](https://github.com/w3c/webcomponents/blob/gh-pages/proposals/html-modules-proposal.md)
-Questions/corrections/feedback are welcome!  I've left TODOs in several places where we still have open questions; any input regarding these is especially appreciated.
+# Proposed spec changes for HTML Modules
 
--- [@dandclark](https://github.com/dandclark), with:\
-&nbsp;&nbsp;&nbsp;&nbsp;[@bocupp](https://github.com/BoCupp) [@samsebree](https://github.com/samsebree) [@travisleithead](https://github.com/travisleithead)
+This is a list of spec areas that will need to be changed to implement our [HTML Modules proposal](https://github.com/w3c/webcomponents/blob/gh-pages/proposals/html-modules-proposal.md).  Questions/corrections/feedback are welcome!  I've left TODOs in several places where we still have open questions; any input regarding these is especially appreciated.
 
-# ES6 Spec Changes ([full spec link](https://tc39.github.io/ecma262/)):
+No changes are proposed to the [ES spec](https://tc39.github.io/ecma262/); HTML Module behavior is defined entirely in HTML5.  Note however that these spec changes are built on top of the proposed ES module refactoring here: https://github.com/tc39/ecma262/pull/1311.  Most notably, HTML Module Record subclasses the Cyclic Module Record introduced in that change.
 
-These spec changes are built on top of the proposed refactoring here: https://github.com/tc39/ecma262/pull/1311
+## HTML5 spec changes ([full spec link](https://html.spec.whatwg.org/)):
```

Comment:
I noticed that a few times in this document the term __HTML5__ is used, I think it would be more accurate to just use __HTML__. See [this note](https://html.spec.whatwg.org/#is-this-html5?).

Reply:
Makes sense.  Addressed in the latest revision.